docs/src/examples.md

```diff
@@ -0,0 +1,42 @@
+# Examples
+
+Here you can find a list of examples to help you quick start using this
+library.
+
+
+# 1D signal
+
+A simple real ``60 Hz`` with the first harmonics:
+
+```julia
+using Plots
+using FFTW
+using DSP
+
+# Number of points
+N = 2^12 - 1
+# Sample spacing
+Ts = 1 / (1.1 * N)
+# Sample rate
+fs = 1 / Ts
+# Start time
+t0 = 0
+tmax = t0 + N * Ts
+
+# time coordinate
+t = t0:Ts:tmax
+
+signal = sin.(2π * 60 * t) + .5 * sin.(2π * 120 * t)
+
+# Fourier Transform of it
+F = fftshift(fft(signal))
+freqs =  fftshift(fftfreq(length(t), fs))
```

Comment:
It could be worthwhile to add a comment explaining why `fftshift` is being used here.